modules/Bio/EnsEMBL/Production/Pipeline/AlphaFold/HiveLoadAlphaFoldDBProteinFeatures.pm

```diff
@@ -19,7 +19,11 @@ HiveLoadAlphaFoldDBProteinFeatures.pm
 
 =head1 DESCRIPTION
 
-This module inserts protein features into an Ensembl core database based on the AlphaFoldDB-UniProt mappings found in the EMBL-EBI AlphaFoldDB SIFTS data and the UniProt-ENSP mappings found in the GIFTS database in order to make the link between AlphaFoldDB and ENSP having a AlphaFoldDB entry as a protein feature for a given ENSP protein.
+This module inserts protein features into an Ensembl core database based on the
+AlphaFoldDB-UniProt mappings found in the EMBL-EBI AlphaFoldDB SIFTS data and
+the UniProt-ENSP mappings found in the GIFTS database in order to make the link
+between AlphaFoldDB and ENSP having a AlphaFoldDB entry as a protein feature for
+a given ENSP protein.
 
 It also populates the "pdb_ens" table in the GIFTS database with similar data.
```

Comment:
@ens-carlos Not sure about this comment, any impact on our GIFTS app then?
@EbiArnie Do we have to update our SOP accordingly to include the GIFTS connection parameters in the registry?

Reply:
I don't think there is any impact on GIFTS as it's just reading from it.